pkg/flag/cache_flags.go

```diff
@@ -7,28 +7,65 @@ import (
 
 	"github.com/samber/lo"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
 	"golang.org/x/xerrors"
 )
 
-const (
-	ClearCacheFlag   = "clear-cache"
-	CacheBackendFlag = "cache-backend"
-	CacheTTLFlag     = "cache-ttl"
-	RedisCACertFlag  = "redis-ca"
-	RedisCertFlag    = "redis-cert"
-	RedisKeyFlag     = "redis-key"
+// e.g. config yaml
+// cache:
+//   clear: true
+//   backend: "redis://localhost:6379"
+//   redis:
+//    ca: ca.pem
+//    cert: cert.pem
+//    key: key
+var (
+	ClearCacheFlag = Flag{
+		Name:       "clear-cache",
+		ConfigName: "cache.clear",
+		Value:      false,
+		Usage:      "clear image caches without scanning",
+	}
+	CacheBackendFlag = Flag{
+		Name:       "cache-backend",
+		ConfigName: "cache.backend",
+		Value:      "",
+		Usage:      "cache backend (e.g. redis://localhost:6379)",
+	}
+	CacheTTLFlag = Flag{
+		Name:       "cache-ttl",
+		ConfigName: "cache.ttl",
+		Value:      "",
```

Comment:
It is time.Duration.

```suggestion
		Value:      time.Duration(0),
```

Reply:
Done